Makefile

```diff
@@ -173,6 +173,25 @@ scorecard: ## Run scorecard test
 	@echo ... Running the scorecard test
 	- operator-sdk scorecard --verbose
 
+##@ Red Hat Certify
+
+bundle:
+	@echo --- Updating the bundle directory with latest yamls from olm-catalog ---
+	rm -rf bundle/*
+	cp -r deploy/olm-catalog/operand-deployment-lifecycle-manager/${CSV_VERSION}/ bundle/
+	cp deploy/olm-catalog/operand-deployment-lifecycle-manager/operand-deployment-lifecycle-manager.package.yaml bundle/
+	zip bundle/operand-deployment-lifecycle-manager bundle/*.yaml
```

Comment:
I can't find where zipped package is used, is this really necessary?

Reply:
When certifying the operator,  we need to upload a zip file with all the YAML files.

Reply:
Add a WIP label, I will use this branch to do preparation works for RH certify